.coveragerc

```diff
@@ -1,3 +1,3 @@
 [run]
 branch = True
-omit = *__init__.py,*wsgi.py,*asgi.py,manage.py,*migrations*,*settings.py,*tests*
\ No newline at end of file
+omit = *__init__.py,*wsgi.py,*asgi.py,manage.py,*migrations*,*settings.py,*tests*,*/usr/lib/python3/dist-packages/*,*/venv/*
```

Comment:
Why was this changed?

Reply:
Should have no impact anyways, so it should be okay.

Reply:
Okay so looked at the action history for this PR, and saw the issues you were having was from Migrations Conflicts, this should now be sorted.